edx_event_bus_kafka/internal/consumer.py

```diff
@@ -213,6 +213,13 @@ def reset_offsets(consumer, partitions):
                     # This log message may be noisy when we are replaying, but hopefully we only see it
                     # once every 30 seconds.
                     logger.info("Offsets are being reset. Sleeping instead of consuming events.")
+
+                    # We are calling poll here because we believe the offsets will not be set
+                    # correctly until poll is called, despite the offsets being reset in a different call.
```

Comment:
Maybe add something in here about how we don't think partitions are assigned until poll is called?